internal/config/profile.go

```diff
@@ -243,6 +244,10 @@ func (p *Profile) Name() string {
 
 func SetName(name string) { Default().SetName(name) }
 func (p *Profile) SetName(name string) {
```

Comment:
Can we also make sure to cover` func (p *Profile) Rename(newProfileName string)`

Reply:
Done